platform/podjs/src/browserPod.ts

```diff
@@ -330,32 +328,83 @@ class IDBPolyOut implements PolyOut {
         throw "Not implemented: writeFile";
     }
 
-    async importArchive(url: string): Promise<string> {
-        const { data: dataUrl, fileName } = FileUrl.fromUrl(url);
-        const blob = await (await fetch(dataUrl)).blob();
+    /// destUrl should be the same as zipId
+    async importArchive(url: string, destUrl?: string): Promise<string> {
+        const { fileName } = FileUrl.fromUrl(url);
+        const buffer = await (await fetch(url)).arrayBuffer();
```

Comment:
OK, you need `cross-fetch` to do this, I guess; since this was not tested before at the unit level, it didn't fail.
You probably made a good choice; however, did you consider node-fetch? It's what's used in some other modules (`remote-pod` IIRC). When we use the same module in two or more places, we "hoist" it to the root so that it's installed only once.

Reply:
I looked at node-fetch, but I need something that works both with node and browsers, without me having to change which function I call based on the platform.